openedx/core/djangoapps/credentials/__init__.py

```diff
@@ -0,0 +1,6 @@
+"""
```

Comment:
@clintonb the functionality of the `credentials`  would be most similar to the `programs` since we will have a `ConfigurationModel` for toggling credentials and some functionality to add program related credentials on learner dashboard so I would suggest that it should remain as a small django app.